src/components/Tipseen/Tipseen.jsx

```diff
@@ -47,6 +47,7 @@ const Tipseen = forwardRef(
     const componentRef = useRef(null);
     const mergedRef = useMergeRefs({ refs: [ref, componentRef] });
     const overrideCloseAriaLabel = closeAriaLabel || TIPSEEN_CLOSE_BUTTON_ARIA_LABEL;
+    const TipseenWrapper = ref || id ? "div" : Fragment;
```

Comment:
I understand this wrapper is useless in general because the ref and the id should be on the tipseen and not on the content linked to it but I can't delete it because backwards compatibility

Reply:
So I convert it to be conditional

Reply:
@hadasfa cool, add the ability to pass an element type